crates/path_resolver/src/lib.rs

```diff
@@ -99,11 +99,11 @@ pub struct ClobberedPath {
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
 struct PathTrieNode {
     /// All tags that touch this prefix *or* any descendant.
-    prefixes: HashSet<PackageName>,
+    prefixes: HashSet<PackageName, RandomState>,
```

Comment:
I think we should either use: 

```suggestion
    prefixes: HashSet<PackageName, ahash::RandomState>,
```

or preferably:

```suggestion
    prefixes: ahash::HashSet<PackageName>,
```

That makes it clear that we are using `ahash`.

Reply:
Done